test/e2e/self_hosted.go

```diff
@@ -48,6 +49,18 @@ type SelfHostedSpecInput struct {
 	SkipCleanup           bool
 	ControlPlaneWaiters   clusterctl.ControlPlaneWaiters
 	Flavor                string
+
+	IncludeUpgrade bool
```

Comment:
```suggestion
	IncludeUpgrade bool
```
TODO: comment + maybe call it `EnableUpgrade`.